csr_check.go

```diff
@@ -127,7 +128,7 @@ func authorizeCSR(machineList *v1beta1.MachineList, req *certificatesv1beta1.Cer
 		}
 		// The CSR requested a DNS name that did not belong to the machine
 		if !foundSan {
-			return false
+			return fmt.Errorf("Missing SAN in DNSNames")
```

Comment:
This error message is not entirely accurate, the error case here is where the CSR has requested a SAN that is not present in the machine's addresses. It might be helpful to print out the CSR and machine SANs also.

Reply:
Done :arrow_down: